crowbar_framework/config/environments/production.rb

```diff
@@ -41,10 +41,10 @@
 
   config.assets.debug = false
   config.assets.raise_runtime_errors = true
-  config.assets.js_compressor = :uglifier
+  config.assets.js_compressor = :closure
   config.assets.css_compressor = :sass
-  config.assets.compile = true
-  config.assets.digest = true
```

Comment:
https://github.com/alexspeller/non-stupid-digest-assets/blob/master/README.md But i would prefer sprockets-redirect referenced on this link as other solutions.